### DISCUSSION TITLE
(more) Alternate backends

### DISCUSSION THREAD

**Discussion by VarLad** [LuxDL/Lux.jl#960] [Ideas]
Would Lux.jl have anything to gain from https://github.com/anicusan/AcceleratedKernels.jl ?

**Comment by avik-pal** [comment]
Currently the way the backend (LuxLib) is written, we do use polyester for CPU and KA for GPU. So there isn't anything performance wise to gain from using AK. That said, it will help us make the code much more concise and I have been meaning to try it out once I have some free time.
